### PR TITLE
Update profile Josh Horton

### PR DESCRIPTION
**Pull request by jthorton**

### CODE REVIEW

**Comment by jthorton** [comment]
Renders fine locally 
image

File: _team/joshhorton.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ---
 name: Josh Horton
 position: Software Scientist
-image_path: https://avatars.githubusercontent.com/u/30259414?v=4
+image_path: https://media.licdn.com/dms/image/v2/C5603AQFbJ8CicIHvfQ/profile-displayphoto-shrink_200_200/profile-displayphoto-shrink_200_200/0/1517285245980?e=1736380800&v=beta&t=r9g4kGKxo62UvYvV42Yqf5mBgDp1XLGzpJYkEdlawzM
 github: jthorton
 scholar: citations?user=qmngJQsAAAAJ&hl=en
 blurb: >-
-  Josh joined OpenFE in October 2024.
+  Josh received his Ph.D from the University of Newcastle in the Cole group. His research is focused on developing
```
Comment (atravitz) [inline _team/joshhorton.md:8]:
```suggestion
  Josh received his Ph.D from the University of Newcastle in the Cole group. His research was focused on developing 
```
do you want past tense here?

Comment (IAlibay) [reply _team/joshhorton.md:8]: How about removing the word altogether and going with "his research focused"

File: _team/joshhorton.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ---
 name: Josh Horton
 position: Software Scientist
-image_path: https://avatars.githubusercontent.com/u/30259414?v=4
+image_path: https://media.licdn.com/dms/image/v2/C5603AQFbJ8CicIHvfQ/profile-displayphoto-shrink_200_200/profile-displayphoto-shrink_200_200/0/1517285245980?e=1736380800&v=beta&t=r9g4kGKxo62UvYvV42Yqf5mBgDp1XLGzpJYkEdlawzM
```
Comment (IAlibay) [inline _team/joshhorton.md:4]: This is a bit low res, does it render fine?

**Review by IAlibay — APPROVED** [review]
Alyssa's suggestion aside this seems good. Might want to check your image after deployment so see if it's not too low res.

File: _team/joshhorton.md (hunk 2 above)
Comment (IAlibay) [reply _team/joshhorton.md:4]: I obviously should have scolled up... sorry about that 😅

File: _team/joshhorton.md (hunk 1 above)
Comment (jthorton) [reply _team/joshhorton.md]: Thanks!

**Pull request merged**